codespell_lib/data/dictionary.txt

```diff
@@ -19150,6 +19153,7 @@ explainatory->explanatory
 explaind->explained
 explanaiton->explanation
 explanaitons->explanations
+explanantion->explanation
```

Comment:
We usually add plural:
```
explanantions->explanations
```

Reply:
@DimitriPapadopoulos If you use the following syntax, your suggestions can be committed directly from the PR interface:

````markdown
```suggestion
explanantion->explanation
explanantions->explanations
```
````

This can usually be injected automatically by clicking the first button to the right of the Write and Preview tabs:

![image](https://github.com/codespell-project/codespell/assets/39996/faa9afe2-fb36-44e7-a0a7-036a7774078a)

and that Markdown will render like this (for people with commit access to the branch, there will be an option to commit the suggestion immediately, or to collect the suggestions into a single commit).

```suggestion
explanantion->explanation
explanantions->explanations
```